backend/de.metas.acct.base/src/main/java/de/metas/acct/gljournal_sap/service/SAPGLJournalService.java

```diff
@@ -82,12 +82,11 @@ public DocStatus getDocStatus(final SAPGLJournalId glJournalId)
 	}
 
 	@NonNull
-	public SAPGLJournal copy(@NonNull final SAPGLJournalCopyRequest copyRequest)
+	public SAPGLJournal reverse(@NonNull final SAPGLJournalCopyRequest copyRequest)
 	{
 		final SAPGLJournal journalToBeCopied = glJournalRepository.getById(copyRequest.getSourceJournalId());
 		final SAPGLJournalCreateRequest createRequest = SAPGLJournalCreateRequest.of(journalToBeCopied,
-																					 copyRequest.getDateDoc(),
-																					 copyRequest.getNegateAmounts());
+																					 copyRequest.getDateDoc());
 
 		return glJournalRepository.create(createRequest, currencyConverter);
```

Comment:
update DocStatus for the source SAPGLJournal via glJournalRepository

Reply:
correction done